app/src/main/res/layout/activity_transforming.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout
```

Comment:
WARNING: [UnusedResources] The resource `R.layout.activity_transforming` appears to be unused
 Unused resources make applications larger and slow down builds.